### PR TITLE
3 login

### LINKED ISSUE #3: Login page
The user should be able to login.

### PR DESCRIPTION
**Pull request by QvistJ**
Closes #3

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class SessionsController < ApplicationController
+  def new
```
Comment (MSaurus) [inline app/controllers/sessions_controller.rb]: Why do you have an empty def)

Comment (itggot-jonathan-lindqvist) [reply app/controllers/sessions_controller.rb]: Because I might use it in the future :)
the guide told me :(

Comment (MSaurus) [reply app/controllers/sessions_controller.rb]: Maybe use rubocop?

Comment (itggot-jonathan-lindqvist) [reply app/controllers/sessions_controller.rb]: It will lint all files, even the default rails files, will have to config it and im too lazy to do it now.

Comment (MSaurus) [reply app/controllers/sessions_controller.rb]: You can use rubocop with rails flag :)
https://docs.rubocop.org/en/latest/cops_rails/#rails

Comment (itggot-jonathan-lindqvist) [reply app/controllers/sessions_controller.rb]: Will add after basic setup is done.

Comment (MSaurus) [reply app/controllers/sessions_controller.rb]: You're way past basic setup ;) but okay, your repo :D

Comment (itggot-jonathan-lindqvist) [reply app/controllers/sessions_controller.rb]: The basic setup includes login, which is pretty basic if you ask me.

Comment (MSaurus) [reply app/controllers/sessions_controller.rb]: So a 7 day project is a basic setup ;) building half of the functionality ain't basic :*

Comment (itggot-jonathan-lindqvist) [reply app/controllers/sessions_controller.rb]: Sorry for learning stuff, at least my tests work. Half? We have discussed all the functionality of this site in person and you know its far from half.

Comment (MSaurus) [reply app/controllers/sessions_controller.rb]: Angry <3

**Pull request merged**